### DISCUSSION TITLE
Where do you look for documentation?

### DISCUSSION THREAD

**Discussion by scionaltera** [scionaltera/agonyforge#156] [Polls]
It's about time to start writing some documentation for this project, both because it's starting to mature a little bit and because it's still fairly complicated to set up a development environment. In other projects I've used the GitHub wiki but I've heard that people don't look there very often. I also don't like that it's all-or-nothing: you can either let everyone edit it or only people with access to the repository (ie. me) can. I'd personally prefer if everyone could do pull requests into it, which feels like a nice middle ground.

**Comment by BurnDivision** [comment]
Setting up a Discord server may be an option.

**Reply by scionaltera** [reply]
Yeah, I'm definitely considering a Discord. I have one for one of my other open source projects and it has been very useful. At some point soon I'll be setting up a public copy of the MUD too, but Discord has the distinct advantage of saving conversations and pinning messages. I feel like the project also needs some actual documentation though.

**Reply by BurnDivision** [reply]
A discord could be great for redirecting people to FAQs in certain read only channels. Then maybe find a Wiki host or something for more in-depth documentation. Discord could also hopefully direct users to that Wiki as well.
